onnxruntime/core/providers/cuda/controlflow/loop.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/providers/cpu/controlflow/loop.h"
+
+namespace onnxruntime {
+class SessionState;
```

Comment:
is this line needed ?

Reply:
No. Will remove.

---
In reply to: [351566539](https://github.com/microsoft/onnxruntime/pull/2444#discussion_r351566539) [](ancestors = 351566539)